.github/workflows/gitlab.yml

```diff
@@ -3,22 +3,30 @@ on:
   pull_request_target:
     types: [labeled]
   push:
+  schedule:
+    # Avoid running on 0 since a lot of other workflows on github start at that
+    # time and this can cause delays or even dropping of jobs:
+    # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+    - cron: 18,48 21-23 * * *
+    - cron: 17,47 0-8 * * *
```

Comment:
is it UTC, like documentation says? If yes, it will be running from 8:00 - 11:00 Lithuanian time, which is not that good. Also will start late. :smile:

Reply:
Good point! I copied the cron config from gitlab and didn't realized that it was set to be in LT timezone. I will move it back by 3 hours

Reply:
also weird configuration of minutes, can't you just start write something like `15-59`

Reply:
The doc linked in the comment explains that a little. Basically github can delay or even drop scheduled workflows when there are too many workflows scheduled at a given time. They specifically mention full hours but as you can imagine round number of minutes (0,15,30,45) probably also are more crowded. To avoid that I've selected those hopefully less crowded minutes to start this workflow.

Reply:
ok, this is kinda reasonable. I probably missunderstood abit, cuz  I tought `schedule: ${{ github.event.schedule != null }}`, this would not allow to queue pipeline if one is already queued